round4/team-01.md

```diff
@@ -0,0 +1,118 @@
+# Chrome Extension Idea: PennCourse Extension
+
+## Authors
+
+Eshaan Chichula, Franci Branda-Chen, Jake Garrison Murphy, Matt fu
+
+## Problem Statement
+
+Our Chrome Extension simplifies the course selection process for Penn students by addressing a specific pain point: the need to switch back and forth between Path@Penn and Penn Course Review to assess course difficulty and instructor quality. By integrating this critical information directly into the Path@Penn interface, our tool minimizes the friction involved in course research, allowing students to make informed decisions with ease.
+
+## Target Audience
+
+This extension is tailored for Penn College students who are navigating the course selection process. 
+
+## Description
+
+Aiming to refine the course selection at Penn, our Chrome Extension introduces a user-friendly popup that condenses information from Penn Course Review, such as course difficulty levels and instructor ratings, and integrates it directly onto the Path@Penn page. This enhancement allows students to effortlessly compare and consider courses within a single browser window.
+
+## Selling Points
+
+1. Integrated Review Popup: Presents a succinct summary of course and instructor evaluations from Penn Course Review in a convenient popup on Path@Penn, streamlining research.
```

Comment:
Potential feature: hyperlink to corresponding PennCourseReview for a given course